packages/kit-bg/src/services/ServiceHistory.ts

```
@@ -1242,7 +1242,7 @@ class ServiceHistory extends ServiceBase {
     });
 
     // refresh BTC fresh address for HD or HW accounts if needed
-    void this.backgroundApi.serviceAccountProfile.syncBTCFreshAddressByAccountId(
+    void this.backgroundApi.serviceFreshAddress.syncBTCFreshAddressByAccountId(
```

ESLint

[error] Insert `{`

(prettier/prettier)